lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -527,7 +527,7 @@ isPending = (== Pending) . (status :: TxMeta -> TxStatus) . snd
 -- layer and that the underlying encoding is rather agnostic to the underlying
 -- backend.
 newtype Address = Address
-    { getAddress :: ByteString
+    { unAddress :: ByteString
```

Comment:
I do like `unAddress` better than `getAddress`.

However there is always the option of name qualification or import hiding if we want to keep the newtype accessors named consistently.